dask/highlevelgraph.py

```diff
@@ -211,6 +147,77 @@ def get_dependencies(self, key: Hashable, all_hlg_keys: Iterable) -> set:
         """
         return keys_in_tasks(all_hlg_keys, [self[key]])
 
+    def __dask_distributed_anno_pack__(self) -> Optional[Mapping[str, Any]]:
+        """Packs Layer annotations for transmission to scheduler
+
+        Callables annotations are fully expanded over Layer keys, while
+        other values are simply transmitted as is
+
+        Returns
+        -------
+        packed_annotations : dict
+            Packed annotations.
+        """
+        if self.annotations is None:
+            return None
+
+        packed = {}
+
+        for a, v in self.annotations.items():
+            if callable(v):
+                packed[a] = {stringify(k): v(k) for k in self}
+                packed[a]["__expanded_annotations__"] = True
+            else:
+                packed[a] = v
+
+        return packed
+
+    @staticmethod
+    def __dask_distributed_anno_unpack__(
```

Comment:
Similar comment here about the method name:

```suggestion
    def __dask_distributed_annotations_unpack__(
```